src/agent/EnvelopeService.ts

```diff
@@ -1,10 +1,19 @@
+import type { Verkey } from 'indy-sdk'
 import { inject, scoped, Lifecycle } from 'tsyringe'
-import { OutboundMessage, UnpackedMessageContext } from '../types'
+
+import { UnpackedMessageContext } from '../types'
 import { Wallet } from '../wallet/Wallet'
 import { ForwardMessage } from '../modules/routing/messages'
 import { AgentConfig } from './AgentConfig'
 import { Logger } from '../logger'
 import { Symbols } from '../symbols'
+import { AgentMessage } from './AgentMessage'
+
+interface Keys {
+  recipientKeys: Verkey[]
+  routingKeys: Verkey[]
+  senderVk: Verkey | null
```

Comment:
Consistency:

```suggestion
  senderVerkey: Verkey | null
```
OR

```suggestion
  senderKey: Verkey | null
```

Reply:
Good catch, updated.